app/views/podcasts/show.rss.builder

```diff
@@ -94,7 +94,10 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
 
           Array(ep.categories).each { |c| xml.category { xml.cdata!(c || '') } }
 
-          xml.itunes :author, @podcast.author_name unless @podcast.author_name.blank?
+          if ep.author_name || @podcast.author_name
```

Comment:
Should these still be checking for `blank?`?  Or would a blank string not make it this far?